GlobalBehaviorandComponents/manageapps.py

```diff
@@ -23,13 +23,19 @@ def gbac_apps():
     okta_admin = OktaAdmin(session[SESSION_INSTANCE_SETTINGS_KEY])
 
     applist = okta_admin.get_applications_by_user_id(user_info["sub"])
+    myapplist = []
+    for app in applist:
+        if "profile" in app:
+            if "createdby" in app["profile"]:
+                if user_info["email"] in app["profile"]["createdby"]:
+                    myapplist.append(app)
```

Comment:
Wouldn't this same logic work? Nested `if` are hard to read.

```
if ("profile" in app) AND ("createdby" in app["profile"]) AND (user_info["email"] in app["profile"]["createdby"]):
    myapplist.append(app)

Reply:
It worked.